examples/docs_snippets/docs_snippets/concepts/partitions_schedules_sensors/partitioned_config_test.py

```diff
@@ -32,4 +32,31 @@ def test_my_partitioned_config():
     assert validate_run_config(do_stuff_partitioned, run_config)
 
 
-# end
+# end_partition_config
+
+# start_partition_keys
+
+
+@daily_partitioned_config(start_date=datetime(2020, 1, 1), hour_offset=3)
+def my_offset_partitioned_config(start: datetime, _end: datetime):
+    return {
+        "ops": {
+            "process_data_for_date": {"config": {"date": start.strftime("%Y-%m-%d")}}
+        }
+    }
+
+
+def test_my_offset_partitioned_config():
+    # test that the first partition is as expected
+    partitions = my_offset_partitioned_config.partitions_def.get_partitions()
+    assert str(partitions[0].value.start) == "2020-01-01T03:00:00+00:00"
+    assert str(partitions[0].value.end) == "2020-01-02T03:00:00+00:00"
+
+    # get a partition for a datetime and assert the output of my_offset_partitioned_config is valid
+    # configutation for the do_stuff_partitioned_job
+
+    run_config = my_offset_partitioned_config.run_config_for_partition_fn(partitions[0])
```

Comment:
is `run_config_for_partition_fn` something we're ok telling users to use?

Reply:
I don't think so, because we don't really expose `Partition` as a public API, and it accepts a `Partition` object

Reply:
hmmm ok - there's a task asking for better docs on how to test PartitionConfigs and make sure they are creating the partitions the user expects. is there a different way to do that? this was the best way i found

Reply:
good question. I think we likely need to add something like a `get_run_config_for_partition_key` method.

I think it makes sense to separate that into a separate PR from this one, because it's not on the critical path of this functionality, if I understand correctly.

Reply:
I agree. Will remove this example and address this in a follow up task